src/LibraryManager.Contracts/FileHelpers.cs

```diff
@@ -210,10 +210,11 @@ public static bool AreFilesUpToDate(FileInfo firstFile, FileInfo secondFile)
         /// Deletes files 
         /// </summary>
         /// <param name="filePaths"></param>
+        /// <param name="rootDirectory"></param>
         /// <returns></returns>
-        public static bool DeleteFiles(IEnumerable<string> filePaths)
+        public static bool DeleteFiles(IEnumerable<string> filePaths, string rootDirectory = null)
```

Comment:
>filePaths [](start = 59, length = 9)

We should assert that all of these are absolute paths

Reply:
Will do

---
In reply to: [201168854](https://github.com/aspnet/LibraryManager/pull/286#discussion_r201168854) [](ancestors = 201168854)